packages/bundler-sdk-viem/src/operations.ts

```diff
@@ -337,6 +338,8 @@ export const populateSubBundle = (
       let { withdrawals, data: simulationStatePostFriendlyReallocation } =
         data.getMarketPublicReallocations(market.id, publicAllocatorOptions);
 
+      let additionalWithdrawals: PublicReallocation[] = [];
```

Comment:
No need to declare it globally right? Can be declared const inside the branch where it is assigned